app/models/field.rb

```diff
@@ -9,22 +9,21 @@ class Field < ApplicationRecord
   serialize :options, Array
 
   validate :belongs_to_form_or_form_draft?
-  validates :data_type,
-            inclusion: { in: DATA_TYPES }
-  validates :number,
-            :prompt,
-            presence: true
+  validates :options, presence: true, if: -> { data_type == 'options' }
+  validates :data_type, inclusion: { in: DATA_TYPES }
+  validates :number, :prompt, presence: true
   # Equivalent to
   # validates :form, uniqueness: { scope: :number, allow_blank: true }
   # but the point is that the number is unique respective to the form,
   # not the other way around
   validates :number, uniqueness: { scope: :form,
-                                   if: -> { form.present? } }
+                                   if: -> { form_id.present? } }
```

Comment:
Curious why this change was needed.

Reply:
I think I was trying to solve a problem, tried 3 things, one thing worked, left the other 2. I took it away and ran the tests and they all passed, so I'm guessing this wasn't actually necessary.